canvas_modules/common-canvas/src/toolbar/toolbar-overflow-item.jsx

```diff
@@ -33,7 +33,6 @@ class ToolbarOverflowItem extends React.Component {
 	render() {
 		const menuItems = this.props.showExtendedMenu ? this.props.generateExtensionMenuItems(this.props.index) : [];
 		const subMenuClassName = this.props.showExtendedMenu ? "" : "toolbar-popover-list-hide";
-
```

Comment:
Still need to resolve this. I know you said in Slack you are working on figuring out what these lines are being added so hopefully you'll get to the bottom of it.